main.go

```diff
@@ -74,15 +74,37 @@ func handleIssueComment(body []byte, gitRepos git.Repos, pullRequests PullReques
 	if !issueComment.IsPullRequest {
 		return SuccessResponse{"Not a PR. Ignoring."}
 	}
-	switch {
-	case isSquashCommand(issueComment.Comment):
+	commentCategory := parseComment(issueComment.Comment)
+	if commentCategory == regularComment {
+		return SuccessResponse{"Not a command I understand. Ignoring."}
```

Comment:
I think `SuccessResponse` is appropriate, yes. I've keeping `ErrorResponse` for cases where e.g. the GitHub API times out and we respond with a 502, or when something unexpected happens in the code. This way, the Webhook view in GitHub settings actually becomes helpful, because you can quickly gauge the bot's health from there, by looking at the number of red triangles, indicating HTTP error responses.

And semantically it's not a failure to satisfy the request but rather a choice not to do anything in that particular case. Doesn't mean that anything has gone wrong.